# HG changeset patch
# User Dan Villiom Podlaski Christiansen <danchr@gmail.com>
# Date 1656840605 -7200
#      Sun Jul 03 11:30:05 2022 +0200
# Branch 1.0.x
# Node ID 5277b791c8e79568832785560cac491d72044534
# Parent  195883d638e0143ab7fb112f8ccba2882a104bb4
tests: handle changed casing in Git 2.37

diff --git a/tests/test-encoding.t b/tests/test-encoding.t
--- a/tests/test-encoding.t
+++ b/tests/test-encoding.t
@@ -33,10 +33,10 @@
   $ . $TESTDIR/latin-1-encoding
   Warning: commit message (did|does) not conform to UTF-8. (re)
   You may want to amend it after fixing the message, or set the config
-  variable i18n.commitencoding to the encoding your project uses.
+  variable i18n.commit[eE]ncoding to the encoding your project uses. (re)
   Warning: commit message (did|does) not conform to UTF-8. (re)
   You may want to amend it after fixing the message, or set the config
-  variable i18n.commitencoding to the encoding your project uses.
+  variable i18n.commit[eE]ncoding to the encoding your project uses. (re)
 
   $ cd ..
   $ git init -q --bare repo.git
